libpod.conf

```diff
@@ -35,6 +35,9 @@ conmon_env_vars = [
 # CGroup Manager - valid values are "systemd" and "cgroupfs"
 cgroup_manager = "systemd"
 
+# Container init binary
+init = "/usr/bin/catatonit"
```

Comment:
Probably best to leave this commented out if we don't expect many people to use it and the hardcoded default is the same

Reply:
Good point. It should also be mentioned in `man libpod.conf`.

Reply:
Done :heavy_check_mark: ... also renamed it to `init_path` and `InitPath` respectively to avoid confusion with the boolean switch.